example/analysis_options.yaml

```diff
@@ -0,0 +1 @@
+include: package:lints/recommended.yaml
```

Comment:
Include a brief 1-liner telling people where they can find more info about this file? dart.dev/go/core-lints (or other)

Reply:
I grabbed the context from the template.